bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractJobWorkerTaskBuilder.java

```diff
@@ -22,11 +22,11 @@
 import io.camunda.zeebe.model.bpmn.instance.zeebe.ZeebeTaskDefinition;
 import io.camunda.zeebe.model.bpmn.instance.zeebe.ZeebeTaskHeaders;
 
-public abstract class AbstractServiceTaskLikeBuilder<
-        B extends AbstractServiceTaskLikeBuilder<B, T>, T extends Task>
+public abstract class AbstractJobWorkerTaskBuilder<
+        B extends AbstractJobWorkerTaskBuilder<B, T>, T extends Task>
```

Comment:
Since we now have this new `JobWorkerTask` construct, I think we should document somewhere what this is.

This `AbstractJobWorkerTaskBuilder` class could be a good place for some javadoc, another option I see is to add some javadoc to `JobWorkerTaskProcessor`. Perhaps you see another better option.